build.zig

```diff
@@ -4,10 +4,10 @@ const builtin = @import("builtin");
 const zls_version = std.SemanticVersion{ .major = 0, .minor = 12, .patch = 0 };
 
 /// document the latest breaking change that caused a change to the string below:
-/// Remove all usages of `std.mem.copy` and remove `std.mem.set` (#18143)
-const min_zig_string = "0.12.0-dev.1767+1e42a3de89";
+/// Move many settings from being per-Compilation to being per-Module (#18160)
+const min_zig_string = "0.12.0-dev.2015+60958d135";
 
-pub fn build(b: *std.build.Builder) !void {
+pub fn build(b: *std.Build) !void {
```

Comment:
The `*std.Build` expression is analyzed before the version check which means that  you will get a `container 'std' has no member called 'Build'` error when building with old zig versions like 0.9.0. The following changes will ensure that the version check is always analyzed first.

```suggestion
const Build = blk: {
    const current_zig = builtin.zig_version;
    const min_zig = std.SemanticVersion.parse(min_zig_string) catch unreachable;
    if (current_zig.order(min_zig) == .lt) {
        @compileError(std.fmt.comptimePrint("Your Zig version v{} does not meet the minimum build requirement of v{}", .{ current_zig, min_zig }));
    }
    break :blk std.Build;
};

pub fn build(b: *Build) !void {
```